sr_hand/scripts/sr_hand/tactile_receiver.py

```diff
@@ -54,26 +54,13 @@ def __init__(self, prefix=""):
                 prefix + "tactile", UBI0All, self.tactile_callback, queue_size=1)
 
     def find_tactile_type(self):
-        try:
-            rospy.wait_for_message(
-                self._prefix + "tactile", ShadowPST, timeout=5.0)
+        message_type = rostopic.get_topic_type("/" + self._prefix + "tactile")[0]
```

Comment:
add **if message_type:**. In case there's no tactile topic - if "string" in None will it throw an exception